package.json

```diff
@@ -9,9 +9,13 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.0.5",
     "react-router-dom": "^6.5.0",
     "react-scripts": "5.0.1",
     "react-svg": "^15.1.16",
+    "redux": "^4.2.0",
+    "redux-logger": "^3.0.6",
+    "uuid": "^9.0.0",
     "web-vitals": "^2.1.4"
```

Comment:
- [ ] As per the project requirements kindly make sure to Add Redux Toolkit(npm install react-redux @reduxjs/toolkit).